elixir/apps/domain/lib/domain/resources/resource/changeset.ex

```diff
@@ -88,15 +121,12 @@ defmodule Domain.Resources.Resource.Changeset do
     changeset
     |> put_default_value(:name, from: :address)
     |> validate_length(:name, min: 1, max: 255)
+    |> put_resource_type()
     |> unique_constraint(:address, name: :resources_account_id_address_index)
     |> unique_constraint(:name, name: :resources_account_id_name_index)
     |> cast_embed(:filters, with: &cast_filter/2)
     |> unique_constraint(:ipv4, name: :resources_account_id_ipv4_index)
     |> unique_constraint(:ipv6, name: :resources_account_id_ipv6_index)
-    |> exclusion_constraint(:address,
-      name: :resources_account_id_cidr_address_index,
-      message: "can not overlap with other resource ranges"
-    )
```

Comment:
@AndrewDryga, I removed this based on one of your commits in this PR, but wanted to double check that it was the correct move.

Reply:
The constraint is not removed but It will only check for overlaps in the CGNAT CIDR ranges. The reason for it is that 0.0.0.0/0 was conflicting with any other address resource.

We need to think through subnet router use cases and actually keep the constraint here and the tests for it.

Reply:
Let's discuss it on standup, a little bit of designing is needed here to make sure that we support subnet router use cases.

Reply:
Sounds good 👍

Reply:
(Just putting this here for documentation) As discussed offline with the team, we'll keep this as it is in the PR for now and reassess in the future if necessary.